stdlib/2and3/mmap.pyi

```diff
@@ -71,6 +70,9 @@ if sys.version_info >= (3,):
         def __setitem__(self, index: int, object: int) -> None: ...
         @overload
         def __setitem__(self, index: slice, object: bytes) -> None: ...
+        # Doesn't actually exist, but the object is actually iterable because it has __getitem__ and
+        # __len__, so we claim that there is also an __iter__ to help type checkers.
```

Comment:
David @euresti, do you have an opinion here? Jelle's reasoning sounds fine to me.

@ilevkivskyi note that there's something here that PEP 544 may not solve (the two ways that "iterable" can be fulfilled). Is this worth doing something about in PEP 544?